WorkflowReactiveSwift/TestingTests/TestingTests.swift

```diff
@@ -0,0 +1,197 @@
+/*
+ * Copyright 2020 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import ReactiveSwift
+import Workflow
+import WorkflowReactiveSwift
+import WorkflowReactiveSwiftTesting
+import WorkflowTesting
+import XCTest
+
+class WorkflowReactiveSwiftTestingTests: XCTestCase {
+    func test_workers() {
+        let renderTester = TestWorkflow()
+            .renderTester(initialState: .init(mode: .worker(input: "otherText"), output: ""))
+
+        renderTester
+            .expect(worker: TestWorker(input: "otherText"))
+            .render { _ in }
+    }
+
+    func test_workerOutput_updatesState() {
+        let renderTester = TestWorkflow()
+            .renderTester(initialState: .init(mode: .worker(input: "otherText"), output: ""))
+
+        renderTester
+            .expect(
+                worker: TestWorker(input: "otherText"),
+                producingOutput: "otherText"
+            )
+            .render { _ in }
+            .verifyState { state in
+                XCTAssertEqual(state, TestWorkflow.State(mode: .worker(input: "otherText"), output: "otherText"))
+            }
+    }
+
+    func test_worker_missing() {
+        let tester = TestWorkflow()
+            .renderTester()
+            .expect(
+                worker: TestWorker(input: "input")
+            )
+
+        expectingFailure(#"Expected child workflow of type: WorkerWorkflow<TestWorker>, key: """#) {
+            tester.render { _ in }
+        }
+    }
+
+    func test_worker_mismatch() {
+        let tester = TestWorkflow()
+            .renderTester(initialState: .init(mode: .worker(input: "test"), output: ""))
+            .expect(
+                worker: TestWorker(input: "not-test")
+            )
+
+        expectingFailures([
+            #"Workers of type TestWorker not equivalent. Expected: TestWorker(input: "not-test"). Got: TestWorker(input: "test")"#,
+        ]) {
+            tester.render { _ in }
+        }
+    }
+
+//    TODO(dhaval): This currently fatals because it's not sure what Rendering to return.
```

Comment:
Ah, right. We can’t actually not expect Workflows and pass tests. Seems fine to remove this.